ci-operator/config/redhat-developer/rhdh/.claude/skills/rhdh-gke-tests/SKILL.md

```
@@ -0,0 +1,75 @@
+
+```bash
+bash "${CLAUDE_SKILL_DIR}/scripts/inspect-gke-cluster.sh" \
+  --project my-project --cluster my-cluster --zone us-east1-b
+```
+
+## Upgrading the Cluster
+
+Upgrades are performed manually via the GCP Console. The inspect script prints
+the direct URL when upgrades are available:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)